### PR TITLE
[13.0][MIG] module base_optional_quick_create

### PR DESCRIPTION
**Pull request by kimirondon**

### CODE REVIEW

File: base_optional_quick_create/__manifest__.py
Comment (llacroix) [inline base_optional_quick_create/__manifest__.py]: Update version to 13

File: base_optional_quick_create/i18n/base_optional_quick_create.pot
Comment (llacroix) [inline base_optional_quick_create/i18n/base_optional_quick_create.pot:7]: Re-export translations in POT for 13.0?

File: base_optional_quick_create/readme/CONTRIBUTORS.rst
Comment (llacroix) [inline base_optional_quick_create/readme/CONTRIBUTORS.rst]: Add yourself as contributor

**Review by llacroix — COMMENTED** [review]
Rebuild the README.rst file the oca-maintainer tools 

https://github.com/OCA/maintainer-tools

**Comment by AaronHForgeFlow** [comment]
It seems commit history is missing. If you have any doubt, just follow the steps here: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch

if you need assistance, just ask for it.

Regards.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
I tested by making product.product model to "Avoid Quick Create". But I can still create products from purchase order lines, for example. It seems not working.

**Comment by kimirondon** [comment]
> It seems commit history is missing. If you have any doubt, just follow the steps here: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch
> 
> if you need assistance, just ask for it.
> 
> Regards.

hello!
Sorry i couldn't answer you before!! thanks a lot for the info.
Regards

**Comment by kimirondon** [comment]
> I tested by making product.product model to "Avoid Quick Create". But I can still create products from purchase order lines, for example. It seems not working.

the right behavior should be raise the product form view when the user tries to create a product from the purchase order lines for example. I will check.
Regards

**Comment by mcassuto** [comment]
@kimirondon any update ?

**Comment by hitrosol** [comment]
I have tried this locally for models : product.product, res.partner and account.account.
It works well 👍

**Review by etobella — CHANGES_REQUESTED** [review]
@kimirondon The commit history has not been maintained, can you review it please????

**Comment by kimirondon** [comment]
@etobella yes i will !

**Comment by kimirondon** [comment]
@etobella i recovered the commit history, let me know if it's ok!

**Comment by etobella** [comment]
It seems right!! Thanks :smile:

**Comment by mcassuto** [comment]
we have this one on production on multiple instances => could we have it merged ?

**Comment by AaronHForgeFlow** [comment]
Still not working on runbot:
image

image

**Comment by mcassuto** [comment]
@kimirondon where is the mismatch ?

**Comment by mcassuto** [comment]
@AaronHForgeFlow hello Aaaon, I would like your feedback on the MR for v14.

**Review by AaronHForgeFlow — APPROVED** [review]
Ok I see the purpose of the module now. It works. Thanks @mcassuto

**Comment by mcassuto** [comment]
@AaronHForgeFlow let's be honnest...
This module works as it and somehow answer the need; it has been approved by customers as well (v13 and v14)...
....
However, I guess you were saying it was NOT working because, like me, you were used to the previous versions where these 2 options (create / create & edit) were simply removed.

I don't know which version did see this change in the behaviour but it has been approved and merged....
@kimirondon did simply migrate the v12 version.

So, what I suggest : 
1/ we merge v13 and v14 now (it could be helpful to a lot of users)
2/ we submit later another version (with a system parameter to enforce the restriction ??)

What do you think ?

**Comment by AaronHForgeFlow** [comment]
I think we can continue with this module as it is. 

I was confused with this other module: https://github.com/OCA/web/tree/13.0/web_m2x_options This one actually removes the " (create / create & edit)" options but not at model level.

**Comment by mcassuto** [comment]
@AaronHForgeFlow thanks ! then what is the next step to have these 2 modules merged ?

**Comment by AaronHForgeFlow** [comment]
Get more approvals including the approval of maintainer of this repository, I am not maintainer of this one.

I see that @etobella  requested changes here regarding the commit history and I am not sure it is fixed. It is kind of weird, the oldest commit and the newest are the same commit?  

First time I see this.

**Comment by llacroix** [comment]
@pedrobaeza any idea who could get that verified to be merged?

**Comment by pedrobaeza** [comment]
Check on the module contributors for pinging them, but you can add your review as well.

**Comment by llacroix** [comment]
@LoisRForgeFlow could you have a look to this PR?

**Review by llacroix — APPROVED** [review]

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
The commit history appear to not be clean, it has migration commits at the beggining and end of history:

image

image

**Comment by llacroix** [comment]
@LoisRForgeFlow anything ?

**Review by llacroix — APPROVED** [review]

**Comment by simahawk** [comment]
@etobella ping

**Comment by llacroix** [comment]
anytime would be great

**Review by acsonefho — APPROVED** [review]

**Comment by kimirondon** [comment]
> The commit history appear to not be clean, it has migration commits at the beggining and end of history:
> 
> image
> 
> image

**Pull request closed without merging**
